content/kubermatic/master/guides/addons/aws_node_termination_handler/_index.en.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "AWS Node Termination Handler"
+date = 2021-01-27T14:07:00+02:00
+weight = 6
+
++++
+
+## AWS Spot Instance
+
+Kubermatic Kubernetes Platform(KKP) is capable of creating different machine types in more than ten different cloud providers.
+One of those providers is AWS. 
+
+AWS cloud provider supports a special machine instances that are called spot instances. Spot instances(according to aws official 
+documentations) are instances that use spare EC2 capacity that is available for less than the On-Demand price.
+Because Spot Instances enable you to request unused EC2 instances at steep discounts, you can lower your Amazon EC2 costs significantly.
+The hourly price for a Spot Instance is called a Spot price.
+
+Kubermatic Kubernetes Platform supports the creation of aws spot instances and use them as worker nodes for any aws user cluster.
+Spot Instances are very good choice in case of cost-effectiveness if you can be flexible about when your applications run and if 
+your applications can be interrupted.
+
+Once a spot instance interruption is announced, a notification will be emitted from aws, stating that a specific spot instance 
+is scheduled for termination two minutes prior the actual termination. AWS node termination handler receives this notification 
+and start the process of node draining for the instance which is being scheduled for the termination.
+
+AWS node termination handler is deployed with any aws user cluster created by KKP to ensure the resilience of the user 
+cluster once the spot instance is interrupted.
+
+## AWS Spot Instances Creation
```

Comment:
Since this is focused on the new clusters, would it make sense to mention that users can create Spot instances in existing clusters as well (via the Create MachineDeployment dialog)?

Reply:
Well it is actually intuitive. The most important thing is, tracking the users attention about this field and what it does, so if a new machine deployment is being, they will see the exact same checkbox